cmd/release-controller/info.go

```diff
@@ -34,6 +34,17 @@ func NewCachingReleaseInfo(info ReleaseInfo, size int64) ReleaseInfo {
 		var err error
 		parts := strings.Split(key, "\x00")
 		switch parts[0] {
+		case "bugs":
+			if strings.Contains(parts[1], "\x00") || strings.Contains(parts[2], "\x00") {
+				s, err = "", fmt.Errorf("invalid from/to")
+			} else {
+				var sArr []string
+				sArr, err = info.Bugs(parts[1], parts[2])
+				if err != nil {
```

Comment:
Why are we caching only in the error case?  I would have expected this to be `if err == nil { s = strings.Join(sArr, "\n") }`

Reply:
That was a typo